frontend/src/metabase/lib/schema_metadata.js

```diff
@@ -470,6 +470,28 @@ const MORE_VERBOSE_NAMES = {
   "greater than or equal to": "is greater than or equal to",
 };
 
+export function doesOperatorExist(operatorName) {
+  return !!FIELD_FILTER_OPERATORS[operatorName];
+}
+
+export function getOperatorByTypeAndName(type, name) {
```

Comment:
"operators" aren't really first-class objects in the sense that there seem to be multiple different representations of what they are within this `schema_metadata` file. The pre-existing `getFilterOperators` is _almost_ useful, but depends on `field` and `table` args, so I've added this new getter function to bag up what operator metadata exists in this file without a dependency on anything besides a given `type` and `name`.